ambari-server/src/main/resources/common-services/HDFS/2.1.0.2.0/package/alerts/alert_metrics_deviation.py

```diff
@@ -484,3 +494,27 @@ def _coerce_to_integer(value):
     return int(value)
   except ValueError:
     return int(float(value))
+
+
+def _get_name_service_by_hostname(hdfs_site, host_name):
+  """
+   Finds the name service which the name node belongs to in an HA or federated setup.
```

Comment:
I think it would make sense to add this function to https://github.com/apache/ambari/blob/trunk/ambari-common/src/main/python/resource_management/libraries/functions/namenode_ha_utils.py